Client/Frontend/Browser/Toolbars/UrlBar/CollapsedURLBarView.swift

```diff
@@ -0,0 +1,94 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import BraveCore
+import BraveShared
+
+/// A view that displays the tab's secure content state and the URL while scrolling into the page
+class CollapsedURLBarView: UIView {
+  private let stackView = UIStackView().then {
+    $0.axis = .horizontal
+    $0.spacing = 4
+    $0.isUserInteractionEnabled = false
+  }
+  
+  private let lockImageView = ToolbarButton(top: true).then {
+    $0.setImage(UIImage(named: "lock_verified", in: .current, compatibleWith: nil)!.template, for: .normal)
+    $0.isHidden = true
+    $0.tintColor = .bravePrimary
+    $0.isAccessibilityElement = true
+    $0.imageView?.contentMode = .center
+    $0.contentHorizontalAlignment = .center
+    $0.accessibilityLabel = Strings.tabToolbarLockImageAccessibilityLabel
```

Comment:
Shouldn't this label be dynamic depending on secure/insecure state?

Reply:
Hmm probably yeah, copy pasted this from URL bar. The string actually reads "Secure connection" so we should probably only set it to that when its actually secure